core/storage/user/gae_models.py

```diff
@@ -110,6 +110,22 @@ class UserSettingsModel(base_models.BaseModel):
     preferred_audio_language_code = ndb.StringProperty(
         default=None, choices=[
             language['id'] for language in constants.SUPPORTED_AUDIO_LANGUAGES])
+    # Displayed name for users on android. They are editable unlike usernames.
```

Comment:
Is this specific to Android or might it be the case on web, as well? I wonder if we should call this 'profile_name'.

Do profiles also have usernames?

Reply:
Yes its specific to android only currently. On web we ask the user to set username and use it as a unique identifier. On android, all profiles can set their name (and edit them) and that is the displayed name there. We don't use username on android.
Yep, `profile_name` sounds better.

Reply:
Done. Used the `real_name` instead to indicate it will be there for full users as well as profiles.

Reply:
Sorry I'm a bit lost now--is this property going to be used specifically for profiles or for everyone? I'm also not keen on 'real name' as a name since we don't actually require that--people are free to use whatever name they'd like to here.

Reply:
I think we should call it profile_name. I think @sajalasati 's point is that even the Administrator profile on Android can rename this name and since administrator profile is a full user, this is being used for both full_users, and profiles.
But since that is actually a profile_name even for the administrator, we can set any android profile name related information in `profile_name`

Reply:
Done.

Reply:
Makes sense--thanks for the context & rename.